CONTRIBUTING.md

```diff
@@ -150,6 +150,11 @@ If the “copy stuff” part consists only of assignments that work for self ass
 
 Doxygen only looks in your .h files; it does not generate documentation from .cpp files. Thus, comments in .cpp files don't need to follow doxygen formatting, and in fact they should not because it is confusing and makes it look like there is API documentation when there isn't. You should mostly use `//`-style comments in .cpp files, and be sure you are addressing your comments to the right audience -- no doxygen reader will ever see them.
 
+We produce two doxygen websites, one for (scripting) users and one for C++
```

Comment:
"website" doesn't seem like the right word here. How about "We produce two sets of doxygen-generated API documentation" or something like that? Whether it gets posted to a web site is a separate issue.